core/load-git.c

```diff
@@ -161,9 +161,9 @@ static duration_t get_duration(const char *line)
 	return d;
 }
 
-static enum dive_comp_type get_dctype(const char *line)
+static enum divemode_t get_dctype(const char *line)
 {
-	for (enum dive_comp_type i = 0; i < NUM_DC_TYPE; i++) {
+	for (enum divemode_t i = 0; i < NUM_DC_TYPE; i++) {
```

Comment:
see my comment in the .h file - this may be what you really wanted (because you didn't want to check against the UNDEF type), but it feels really ugly to me if that's what you are trying to do (and an error otherwise)